### PR TITLE
feature#33 mobile layout for form employees

### PR DESCRIPTION
**Pull request by carlos-ssh**
- Create views for employees that displays it's data.
- Create views for employee form with style.

### CODE REVIEW

**Review by EdwardMD — DISMISSED** [review]
great job.

**Review by ever864 — APPROVED** [review]
Buen avance con el formulario, buen trabajo!

**Review by EdwardMD — APPROVED** [review]
EXCELLENT, closer to complete this project.

File: app/assets/stylesheets/layouts/_employeeform.scss [hunk 1]
```diff
@@ -0,0 +1,141 @@
+.form-container {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
```
Comment (juperameza) [inline app/assets/stylesheets/layouts/_employeeform.scss:4]: You should alphabetically order your styles

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by HikingCarrot7 — APPROVED** [review]
LGTM!

File: app/assets/stylesheets/layouts/_employeeform.scss (hunk 1 above)
Comment (ever864) [reply app/assets/stylesheets/layouts/_employeeform.scss:4]: How I can resolve it?

Comment (germanruzca) [reply app/assets/stylesheets/layouts/_employeeform.scss:4]: Sort them, if you use VSCode you can type `CTRL + P` or `CMD + P` and choose the Sort Lines Ascending. Affect the lines that are select

Comment (juperameza) [reply app/assets/stylesheets/layouts/_employeeform.scss:4]: Exactly like German said 
Do it on all the styles is a good practice:)

Comment (ever864) [reply app/assets/stylesheets/layouts/_employeeform.scss:4]: thanks @germanruzca and @juperameza

**Review by ever864 — APPROVED** [review]
Buen trabajo!

**Review by juperameza — APPROVED** [review]
LGTM

**Pull request merged**